packages/opentelemetry-propagator-jaeger/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export * from './context/propagation/JaegerHttpTraceFormat.js';
```

Comment:
Can we keep `JaegerTraceFormat` in `src/` dirrectly, why do we need to have sub-folder?

Reply:
Also, this should be just `export * from './context/propagation/JaegerHttpTraceFormat';` or `export * from './JaegerHttpTraceFormat';`

Reply:
@mayurkale22 i did it like in opentelemetry-core to keep the same structure. I don't know how better, because this is my first experience with this technology stack. In other languages like Java, Scala, Kotlin and etc. we keeping the same structure for the same responsibilities.

Reply:
In node it is very common to have a package with a single file in no folder structure of any kind if it only fulfills a single purpose.